lib/qa/authorities/linked_data/config.rb

```diff
@@ -0,0 +1,70 @@
+require 'qa/authorities/linked_data/config/config_merge'.freeze
+require 'qa/authorities/linked_data/config/term_config'.freeze
+require 'qa/authorities/linked_data/config/search_config'.freeze
+require 'json'
+
+module Qa::Authorities
+  module LinkedData
+    class Config
+      attr_reader :authority_name
+      attr_reader :authority_config
+
+      # Initialize to hold the configuration for the specifed authority.  Configurations are defined in config/authorities/linked_data.  See README for more information.
+      # @param [String] the name of the configuration file for the authority
+      # @return [Qa::Authorities::LinkedData::Config] instance of this class
+      def initialize(auth_name)
+        @authority_name = auth_name
+        auth_config
+      end
+
+      class << self
```

Comment:
Instead  of doing this, can you just `extend Qa::Authorities::LinkedData::ConfigMerge`  Please provide some documentation about what is being included and for what purpose?  I don see `merge` being called anywhere in this file. Can you help me understand why this is needed here?